ArchiveCompress task unable to read from Kestra's internal storage

### Expected Behavior

ArchiveCompress task should be able to archive compress a file from Kestra's internal storage.

### Actual Behaviour

On providing the file from Kestra's internal storage, the job errors out with the following error:

```
2024-02-17 17:00:57.285 Unrecognized token 'kestra': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"kestra:///company/team/compression-1/executions/1lwBXDGJKAY9BVoBZD1wOh/tasks/httpdownload/43tZi7k6UHxj5Z9oxUz49L/17747190915210794141.csv"; line: 1, column: 7]
2024-02-17 17:00:57.285 com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'kestra': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"kestra:///company/team/compression-1/executions/1lwBXDGJKAY9BVoBZD1wOh/tasks/httpdownload/43tZi7k6UHxj5Z9oxUz49L/17747190915210794141.csv"; line: 1, column: 7]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:2477)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:760)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:3041)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:2082)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:808)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4912)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4818)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3772)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3755)
	at io.kestra.plugin.compress.ArchiveCompress.writeArchive(ArchiveCompress.java:91)
	at io.kestra.plugin.compress.ArchiveCompress.run(ArchiveCompress.java:74)
	at io.kestra.plugin.compress.ArchiveCompress.run(ArchiveCompress.java:27)
	at io.kestra.core.runners.Worker$WorkerThread.run(Worker.java:729)
```

### Steps To Reproduce

1. Use the following flow:

```
id: compression-1
namespace: company.team
tasks:
  - id: httpdownload
    type: io.kestra.plugin.fs.http.Download
    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/products.csv"
  - id: archive_compress
    type: "io.kestra.plugin.compress.ArchiveCompress"
    from: "{{ outputs.httpdownload.uri }}"
    algorithm: ZIP
    compression: GZIP
```

2. Execute the flow.

### Environment Information

- Kestra Version: 0.14.4
- Plugin version: 0.14.4
- Operating System (OS / Docker / Kubernetes): Docker
- Java Version (If not docker): N/A


### Example flow

```
id: compression-1
namespace: company.team
tasks:
  - id: httpdownload
    type: io.kestra.plugin.fs.http.Download
    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/products.csv"
  - id: archive_compress
    type: "io.kestra.plugin.compress.ArchiveCompress"
    from: "{{ outputs.httpdownload.uri }}"
    algorithm: ZIP
    compression: GZIP
```

I tried the following flow as suggested in one of the examples:

```
id: compression-1
namespace: company.team
tasks:
  - id: products_download
    type: io.kestra.plugin.core.http.Download
    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/products.csv"

  - id: orders_download
    type: io.kestra.plugin.core.http.Download
    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/orders.csv"

  - id: archive_compress
    type: "io.kestra.plugin.compress.ArchiveCompress"
    from:
      products.csv: "{{ read(outputs.products_download.uri) }}"
      orders.csv: "{{ read(outputs.orders_download.uri) }}"
    algorithm: ZIP
    compression: GZIP
```

Still getting error:
```
Caused by: java.net.URISyntaxException: Illegal character in path at index 46: product_id,product_name,product_category,brand
 1,streamline turn-key systems,Electronics,gomez
 2,morph viral applications,Household,wolfe
 3,expedite front-end schemas,Household,davis-martinez
 4,syndicate robust ROI,Outdoor,ruiz-price
 5,optimize next-generation mindshare,Outdoor,richardson
 6,scale end-to-end metrics,Outdoor,thompson
 7,implement wireless e-services,Clothing,gomez
 8,e-enable seamless bandwidth,Outdoor,horton-morris
 9,optimize mission-critical info-mediaries,Books,cox-taylor
 10,deploy clicks-and-mortar architectures,Household,horton-morris
 11,strategize wireless architectures,Books,harris
 12,architect transparent solutions,Household,king-navarro
 13,orchestrate visionary deliverables,Books,gomez
 14,utilize next-generation methodologies,Clothing,bowman
 15,exploit global markets,Outdoor,thompson
 16,strategize world-class eyeballs,Books,thompson
 17,synthesize customized e-tailers,Clothing,harris
 18,deliver integrated interfaces,Clothing,lewis
 19,monetize B2B ROI,Books,crawford-gaines
 20,envisioneer cross-media convergence,Electronics,wolfe
at java.base/java.net.URI$Parser.fail(Unknown Source)
at java.base/java.net.URI$Parser.checkChars(Unknown Source)
at java.base/java.net.URI$Parser.parseHierarchical(Unknown Source)
at java.base/java.net.URI$Parser.parse(Unknown Source)
at java.base/java.net.URI.&lt;init&gt;(Unknown Source)
... 6 more
```

--- a/src/main/java/io/kestra/plugin/compress/ArchiveCompress.java
+++ b/src/main/java/io/kestra/plugin/compress/ArchiveCompress.java
@@ -35,18 +35,46 @@ import jakarta.validation.constraints.NotNull;
 @Plugin(
     examples = {
         @Example(
-            code = {
-                "from:",
-                "  myfile.txt: \"{{ inputs.files }}\"",
-                "algorithm: ZIP",
-            }
+            full = true,
+            code = """
+                id: archive_compress
+                namespace: company.team
+
+                inputs:
+                  - id: file
+                    type: FILE
+
+                tasks:
+                  - id: "archive_compress"
+                    type: "io.kestra.plugin.compress.ArchiveCompress"
+                    from:
+                      myfile.txt: "{{ inputs.file }}"
+                    algorithm: ZIP
+                """
         ),
         @Example(
-            code = {
-                "from: \"{{ outputs.taskId.uri }}\"",
-                "algorithm: TAR",
-                "compression: GZIP"
-            }
+            full = true,
+            code = """
+                id: archive_compress
+                namespace: company.team
+
+                tasks:
+                  - id: products_download
+                    type: io.kestra.plugin.core.http.Download
+                    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/products.csv"
+
+                  - id: orders_download
+                    type: io.kestra.plugin.core.http.Download
+                    uri: "https://raw.githubusercontent.com/kestra-io/datasets/main/csv/orders.csv"
+                
+                  - id: archive_compress
+                    type: "io.kestra.plugin.compress.ArchiveCompress"
+                    from:
+                      products.csv: "{{ outputs.products_download.uri }}"
+                      orders.csv: "{{ outputs.orders_download.uri }}"
+                    algorithm: TAR
+                    compression: GZIP
+                """
         )
     }
 )

--- a/src/main/java/io/kestra/plugin/compress/ArchiveDecompress.java
+++ b/src/main/java/io/kestra/plugin/compress/ArchiveDecompress.java
@@ -36,11 +36,23 @@ import java.util.Map;
 @Plugin(
     examples = {
         @Example(
-            code = {
-                "from: \"{{ inputs.files }}\"",
-                "algorithm: ZIP",
-                "compression: GZIP"
-            }
+            full = true,
+            code = """
+                id: archive_decompress
+                namespace: company.team
+
+                inputs:
+                  - id: file      
+                    description: Compressed file
+                    type: FILE
+
+                tasks:
+                  - id: archive_decompress
+                    type: io.kestra.plugin.compress.ArchiveDecompress
+                    from: "{{ inputs.file }}"
+                    algorithm: ZIP
+                    compression: GZIP
+                """
         )
     }
 )

--- a/src/main/java/io/kestra/plugin/compress/FileCompress.java
+++ b/src/main/java/io/kestra/plugin/compress/FileCompress.java
@@ -28,10 +28,22 @@ import jakarta.validation.constraints.NotNull;
 @Plugin(
     examples = {
         @Example(
-            code = {
-                "from: \"{{ inputs.files }}\"",
-                "compression: Z"
-            }
+            full = true,
+            code = """
+                id: file_compress
+                namespace: company.team
+                
+                inputs:
+                  - id: file  
+                    description: File to be compressed
+                    type: FILE
+                
+                tasks:
+                  - id: compress
+                    type: io.kestra.plugin.compress.FileCompress
+                    from: "{{ inputs.file }}"
+                    compression: Z
+                """
         )
     }
 )

--- a/src/main/java/io/kestra/plugin/compress/FileDecompress.java
+++ b/src/main/java/io/kestra/plugin/compress/FileDecompress.java
@@ -27,10 +27,22 @@ import jakarta.validation.constraints.NotNull;
 @Plugin(
     examples = {
         @Example(
-            code = {
-                "from: \"{{ inputs.files }}\"",
-                "compression: Z"
-            }
+            full = true,
+            code = """
+                id: file_decompress
+                namespace: company.team
+                
+                inputs:
+                  - id: file  
+                    description: File to be decompressed
+                    type: FILE
+                
+                tasks:
+                  - id: decompress
+                    type: io.kestra.plugin.compress.FileDecompress
+                    from: "{{ inputs.file }}"
+                    compression: Z
+                """
         )
     }
 )

